.github/workflows/deploy-server.yml

```diff
@@ -0,0 +1,69 @@
+name: Deploy to Amazon ECS
+
+on:
+  pull_request: # for now, trigger this on PR so I can test it before merging
```

Comment:
:stare:

Reply:
The other option is pushing to `master` :badpokerface: